src/main/java/com/koichi/assignment8/StudentMapper.java

```diff
@@ -7,7 +7,31 @@
 
 @Mapper
 public interface StudentMapper {
+    @Select("SELECT *  from students " +
+            "WHERE schoolyear IN ('1年生')")
+    List<Student> findAll_firstgrade();

```

Comment:
命名規則は徹底しましょう。

Javaのコード上はキャメルケース
DB上はスネークケースが望ましいです。
```suggestion
    List<Student> findAllFirstGrade();

```

Reply:
3学年それぞれメソッドを用意するのでは無く、gradeを引数として受け取って
`findByGrade`という一つのメソッドで処理しましょう。

Reply:
また、`school_year`カラムには"年生"という文字列は不要かもしれませんね。
ただの数字として受け取ってしまっても良いかと！

Reply:
- カラムの"年生"の削除
   e2f2c4cfb0f17c369b3fcf6b02f78567c60d89f4